pkg/clusterregistryconfig/flags_test.go

```diff
@@ -37,12 +38,12 @@ var _ = Describe("Cluster Registry Config tests", func() {
 				Equal("-----BEGIN CERTIFICATE-----\n/abc\n-----END CERTIFICATE-----"))
 
 		})
-		It("KO: fail if the spec file is invalid", func() {
+		It("OK: fail if the spec file is invalid", func() {
```

Comment:
Oh @patrickjennings it's not a typo. KO is for tests with expected failures while OK is for best case scenario. No need to change those.

Reply:
Sorry about that. "KO", nice convention! Reverted those changes.

Reply:
Np! Also it's conventional to have only one commit per each change. CS side has the option to squash the commits but I don't think rosa has it.
You can do 
```
git rebase -i HEAD~2
```
comment the 2nd commit
And then force push the PR.

Reply:
Gotcha. Rebased and squashed the commits.

Reply:
Perfect, thanks @patrickjennings